Authentication/authentication.css

```diff
@@ -0,0 +1,83 @@
+:root {
+    --body-bg-color:rgb(243, 243, 243);
+    --page-bg-color :white;
+    --gray-color:gray;
+}
+
+body {
+    font-family: sans-serif;
+    background-color: var(--body-bg-color);
+    margin: 0;
+    padding: 0;
+}
+
+.centered {
+    text-align: center;
+}
+
+.flex-page {
+    display: flex;
+}
+
+.page-wrapper {
+    margin: 3rem auto;
+    background-color: var(--page-bg-color);
+    flex-direction: column;
+    width: 24rem;
+    padding: 2rem;
+    border-radius: 0.5rem;
+}
+
+.page-heading {
+    margin-top: 0;
+    margin-bottom: 1rem;
+}
+
+.input-labels {
+    margin-bottom: 0.5rem;
+}
+
+.input-field {
+    height: 1.8rem;
+    border: 1px solid var(--gray-color);
+    border-radius: 3px;
+    outline: none;
+    margin-bottom: 1rem;
+    padding-left: 5px;
+}
+
+.password-info {
+    align-items: center;
+    margin-bottom: 1rem;
+}
+
+input[type="checkbox"] {
+    transform: scale(1.4);
```

Comment:
just a small UX suggestion instead of putting the scale(1.4) put it as scale(1.08) that would be very subtle, and also give transition to the input element to make the animation works smoothly.